test/grouping.jl

```diff
@@ -2,14 +2,97 @@ module TestGrouping
     using Base.Test
     using DataTables
 
+    srand(1)
     dt = DataTable(a = repeat([1, 2, 3, 4], outer=[2]),
                    b = repeat([2, 1], outer=[4]),
                    c = randn(8))
     #dt[6, :a] = Nullable()
     #dt[7, :b] = Nullable()
 
-    cols = [:a, :b]
+    nullfree = DataTable(Any[collect(1:10)], [:x1])
+    # colwise(::Vector{<:Function}, ::AbstractDataTable)
+    fxn = [sum]
```

Comment:
As the tests are structured it serves both as a header for the test group and so it can be reused to test `size(cw) == (length(fxn),ncol(dt))`. If it really bothers you you're welcome to change it, but I really don't see why this is an issue